contracts/TopShotShardedCollection.cdc

```diff
@@ -37,19 +37,20 @@
 
 import NonFungibleToken from 0xNFTADDRESS
 import TopShot from 0xTOPSHOTADDRESS
+import ViewResolver   from 0xVIEWRESOLVERADDRESS
 
-pub contract TopShotShardedCollection {
+access(all) contract TopShotShardedCollection {
 
     // ShardedCollection stores a dictionary of TopShot Collections
     // A Moment is stored in the field that corresponds to its id % numBuckets
-    pub resource ShardedCollection: TopShot.MomentCollectionPublic, NonFungibleToken.Provider, NonFungibleToken.Receiver, NonFungibleToken.CollectionPublic { 
+    access(all) resource ShardedCollection: TopShot.MomentCollectionPublic, NonFungibleToken.Provider, NonFungibleToken.Receiver, NonFungibleToken.CollectionPublic, ViewResolver.ResolverCollection {
         
         // Dictionary of topshot collections
-        pub var collections: @{UInt64: TopShot.Collection}
+        access(all) var collections: @{UInt64: TopShot.Collection}
```

Comment:
this needs to be `access(self)` or `access(contract)`. If it was public, anyone with a reference from a public capability would be able to cast this to its concrete type and access this field